### PR TITLE
419 admin availability dates

### LINKED ISSUE #419: [FRONTEND] Make admin availability display the available dates
**Is your feature request related to a problem? Please describe.**

Right now on the availability page it displays "open" for available dates (shown below)

You will have to edit `client\src\components\composite\Admin\AdminAvailabilityView\AdminAvailabilityView.tsx`

Image

Instead, we want it to actually show the availabilities:

Image

It will just involve doing something similar to #288 

You should also replace the manual timestamp conversions with the function `timestampToDate` in `client\src\components\utils\Utils.tsx`

**BEFORE MERGING**

- [ ] Storybooks created where possible
- [ ] Tested with [nock](https://tanstack.com/query/v3/docs/framework/react/guides/testing) if data fetching
- [ ] Tests written for critical interactions
- [ ] PR Reviewed (For non-trivial changes)
- [ ] Changes tested after rebasing on master or merging in master (_hint_: `git fetch origin master:master`, then `git rebase master` or `git merge master`)
- [ ] All required PR checks passing

### PR DESCRIPTION
**Pull request by zlrkw11**
closes #419

### CODE REVIEW

**Comment by zlrkw11** [comment]
image
- available spaces / max spaces

**Review by bcho892 — APPROVED** [review]
Gj!

File: client/src/components/composite/Admin/AdminAvailabilityView/AdminAvailabilityView.tsx [hunk 1]
```diff
@@ -123,23 +124,23 @@ const AdminAvailabilityView = ({
...
-            tileContent={({ date }) =>
-              // Find slots that are "available"
-              slots.some(
+            tileContent={({ date }) => {
+              const slot = slots.find(
+                // Find slots that are "available"
                 (slot) =>
-                  new Date(slot.date.seconds * MS_IN_SECOND).toDateString() ===
+                  timestampToDate(slot.date).toDateString() ===
                     date.toDateString() && slot.maxBookings > 0
```
Comment (bcho892) [inline client/src/components/composite/Admin/AdminAvailabilityView/AdminAvailabilityView.tsx:135]: Can we actually remove slot.maxbookings > 0

Comment (zlrkw11) [reply client/src/components/composite/Admin/AdminAvailabilityView/AdminAvailabilityView.tsx:135]: resolved

**Pull request merged**